dtd/locales.dtd

```diff
@@ -17,4 +17,5 @@
 		iso639-3 CDATA #REQUIRED
 		complete (true | false) #IMPLIED
 		stylesheet CDATA #IMPLIED
-		direction CDATA #IMPLIED>
+		direction CDATA #IMPLIED
+		lastFirst (true | false) #IMPLIED>
```

Comment:
Shouldn't that be "family name first" rather than "last name first"? (e.g. `familyFirst`)